cartography/intel/aws/ec2/vpc_peerings.py

```diff
@@ -0,0 +1,173 @@
+import logging
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_vpc_peerings(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    return client.describe_vpc_peering_connections()['VpcPeeringConnections']
+
+
+@timeit
+def load_vpc_peerings(neo4j_session, data, region, aws_account_id, aws_update_tag):
+
+    ingest_vpc_peerings = """
+    UNWIND {vpc_peerings} AS vpc_peering
+
+    MERGE (pcx:PeeringConnection:AWSPeeringConnection{id: vpc_peering.VpcPeeringConnectionId})
```

Comment:
Is it possible to attach `pcx` to `avpc` and then `avpc` to `aaccount` etc etc in this query? I think we should try to make sure AWS resource nodes are connected to their accounts as early as possible.

Reply:
I hope I managed to fix this with the last commit #6dc5950